merlin/models/tf/blocks/core/transformations.py

```diff
@@ -67,11 +67,11 @@ def call(self, inputs: TabularData, **kwargs) -> TabularData:
                 row_lengths = val[1][:, 0]
                 ragged = tf.RaggedTensor.from_row_lengths(values, row_lengths)
                 if self.max_seq_length:
-                    outputs[name] = ragged.to_tensor(shape=[None, self.max_seq_length])
+                    outputs[name] = tf.squeeze(ragged.to_tensor(shape=[None, self.max_seq_length]))
```

Comment:
Do we have to check the num dims herte to know whether to squeeze or not?

Reply:
Yeah. If batch size or `self.max_seq_length` are 1 we might have an issue with tf.squeeze here